routes/web.php

```diff
@@ -13,10 +15,18 @@
 |
 */
 
-Route::get('/', function () {
-    return view('user.welcome');
-});
+Route::get('/', [Controller::class, 'index']);
```

Comment:
```suggestion
Route::get('/', 'Controller@index');
```
Có thể viết ngắn gọn như sau, ko cần `use App\Http\Controllers\Controller;` bên trên nữa